web/satellite/src/views/registration/RegisterArea.vue

```diff
@@ -305,7 +300,6 @@ import LogoWithPartnerIcon from '@/../static/images/partnerStorjLogo.svg';
 import LogoIcon from '@/../static/images/logo.svg';
 import SelectedCheckIcon from '@/../static/images/common/selectedCheck.svg';
 import BottomArrowIcon from '@/../static/images/common/lightBottomArrow.svg';
-import RegisterGlobe from '@/../static/images/register/RegisterGlobe.svg';
```

Comment:
could you also delete this svg from static folder please? I'm pretty sure it's unused from now